WhisperAPI/WhisperAPI/Services/SuggestionsService.cs

```diff
@@ -40,6 +60,22 @@ public IEnumerable<SuggestedDocument> GetSuggestions(string querry)
             return documents;
         }
 
+        private bool IsIntentRelevant(NlpAnalysis nlpAnalysis)
+        {
+            foreach (var intent in nlpAnalysis.Intents)
```

Comment:
- Can't this be replaced by something like : `!nlpAnalysis.Intents.contains(this._intents)` ?
- Why do we check each (var intent in nlpAnalysis.Intents) ? Will we discard a querry if there is "need help" : 98% and "greeting" : 2% ?

Reply:
I agree with you about the discard. However, i think it's gonna be in another release or we can decide now what we want there